### DISCUSSION TITLE
Alpha version for 1.0.0

### DISCUSSION THREAD

**Discussion by shaunabanana** [shaunabanana/intrigue#15] [Announcements]
This version is a preview of the upcoming 1.0.0 release. Please note that features such as importing old documents are not complete yet. There may be bugs hidden somewhere, so please be careful when using it for actual work.

## Feature highlights:
* **Offline-first distributed collaborative editing** powered by SyncedStore (Yjs behind the scenes). Create a web link that you can share to others and let them collaborate in real time. There's no central server. **You keep and distribute your own data.**
* **Zotero Quick Copy translator** allows you to copy an item in Zotero, and paste it in a new Intrigue note to create a Reference. Upon double-clicking, it will lead you back to the item in Zotero. You can download the translator here.
* **URL metadata fetching** are now back. Paste in a web link, and it will now again be correctly turned blue.
* **Undo/redo** is now supported. No more worrying that you may make a mistake by deleting a node.

## Upcoming for 1.0.0
* **Import and upgrade files** created by older Intrigue versions.
* **Export** to BibTeX, HTML, and various other formats.

## Future plans for 1.x:
* **Deep linking support** lets you copy a sharable link to not only the document, but any element in the canvas.

## Known issues
* Text notes that contain a URL among the text will be turned into a reference, thereby loosing the text content.

<hr /><em>This discussion was created from the release <a href='https://github.com/shaunabanana/intrigue/releases/tag/v1.0.0-alpha'>Alpha version for 1.0.0</a>.</em>

**Comment by mageedoo** [comment]
can you show how to install it? which file to download and install?
I download the SETUP file .exe 
but the window has not saving option. and the share option is not clickable

**Reply by shaunabanana** [reply]
Hi @mageedoo ! Unfortunately I haven't been able to test it on a real Windows machine. In the next couple of weeks I should be able to find some time (and a PC) to test this issue. In the mean time, please feel free to [file an issue here](https://github.com/shaunabanana/intrigue/issues) and include any screenshots/information so that I can look into this☺️

**Reply by mageedoo** [reply]
Thank you @shaunabanana , in my case I am using windows and the share bottom is not working, but I found If I press ( Ctrl + S ) the file will be saved in the driver and can be opened again

**Reply by shaunabanana** [reply]
I just published a [new release](https://github.com/shaunabanana/intrigue/releases/tag/v1.0.0-alpha2) which fixes this problem, please check it out :)

**Comment by mageedoo** [comment]
It works
it is also more stable than the previous one

thanks alot

**Comment by mageedoo** [comment]
Hello my brother. I am facing a big problem as I am preparing for my PHD
proposal exam
all the data in an intrigue file now are gone I cannot see them
see the attached file
it very urgent
can you please help me if there is any way to do it?

**Reply by shaunabanana** [reply]
Hi @mageedoo ! I'm very sorry to hear this. Unfortunately, it seems that Github doesn't support attaching files via email. Could you directly send me the file at shengchenzhang1207@gmail.com so that I can take a look?

**Comment by mageedoo** [comment]
it seems like the file is damaged

**Comment by mageedoo** [comment]
thank you brother
I sent the file to your other email

**Comment by mageedoo** [comment]
Did you receive the file?
Is there any hope to restore it ?

Thanks
